engine/polyglot-api/src/main/java/org/enso/polyglot/debugger/IdExecutionService.java

```diff
@@ -52,7 +52,7 @@ public interface Callbacks {
      * @return {@code null} should the execution of the node be performed; any other value to skip
      *     the execution and return the value as a result.
      */
-    Object findCachedResult(Info info);
+    Object findCachedResult(Info info, UUID downstreamDependency);
```

Comment:
- I tried hard to untangle instrumentation changes with `RuntimeCache` & co.
- thus I'd like to deeply understand changes to this interface
- so far the changes here are surprising to me
- the only one that I envision here is `readingLocalVariable(someId)`
- that's a currently missing information from instrumentation to implement reactiveness